storage/disk/disk.go

```diff
@@ -942,3 +932,25 @@ func dataDir(dir string) (string, error) {
 
 	return filepath.Join(dir, "data"), nil
 }
+
+func createSymlink(target, symlink string) error {
+	var lerr error
+
+	if _, lerr = os.Lstat(symlink); lerr == nil {
+		if err := os.Remove(symlink); err != nil {
+			return err
+		}
+
+		if err := os.Symlink(target, symlink); err != nil {
+			return err
+		}
+	} else if errors.Is(lerr, os.ErrNotExist) {
+		if err := os.Symlink(target, symlink); err != nil {
+			return err
+		}
+
+		lerr = nil
```

Comment:
Can't we just `return nil` here? 🤷 It's fine

Reply:
We can 😄 . Updated.